crates/pixi_config/src/lib.rs

```diff
@@ -909,6 +970,23 @@ impl Config {
             .validate()
             .map_err(|e| ConfigError::ValidationError(e, path.to_path_buf()))?;
 
+        // check proxy config
+        if config.proxy_config.https.is_none() && config.proxy_config.http.is_none() {
+            if !config.proxy_config.non_proxy_hosts.is_empty() {
+                tracing::warn!("proxy_config.non_proxy_hosts is not empty but will be ignored")
```

Comment:
```suggestion
                tracing::warn!("proxy-config.non-proxy-hosts is not empty but will be ignored, as no https or http config is set.")
```
We use the kebab-case for user facing prompts, and I would like to add a reason for ignoring it.